worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -399,6 +403,7 @@ private static class SchematicSaveTask extends SchematicOutputTask<Void> {
         public Void call() throws Exception {
             try {
                 writeToOutputStream(new FileOutputStream(file));
+                WorldEdit.getInstance().getSchematicsManager().update();
```

Comment:
Why aren't we passing `file` down here? There's no need to do a full rescan when we could just forcibly add the appropriate file entry.